content/docs/reactivesearch/react-searchbox/apireference.md

```diff
@@ -134,33 +135,9 @@ Example uses of searchbox UI:
 
     > See impact of aggregationField with these example for [React](/docs/reactivesearch/v3/advanced/groupingresults#how).
 
--   **analytics** `boolean` [optional]
-    records search and click analytics when set to true and appbase.io is used as a backend. Defaults to false.
-    Search analytics get recorded with no code changes required. For recording click analytic, you have to call the `triggerClickAnalytics` function (inside `render`) by using the `_click_id` property of the data item as an argument.
-
--   **analyticsConfig** `object` [optional]
-    Set the `analyticsConfig` prop to customize the analytics experience when appbase.io is used as a backend. It accepts an object which has the following properties:
-
-    -   **emptyQuery** `Boolean` If `false` then appbase.io will not record the analytics for the empty queries i.e `match_all` queries. Defaults to `true`.
-    -   **suggestionAnalytics** `Boolean` If `false` then appbase.io will not record the click analytics for the suggestions. Defaults to `true`.
-    -   **userId** `String` It allows you to define the user id which will be used to record the appbase.io analytics.
-    -   **customEvents** `Object` It allows you to set the custom events which can be used to build your own analytics on top of the appbase.io analytics. Further, these events can be used to filter the analytics stats from the appbase.io dashboard.
-        <br/>
-        For example in the following code, we're setting up two custom events that will be recorded with each search request.
-
-    ```js
-    <SearchBox
-    	analytics
-    	analyticsConfig={{
-    		customEvents: {
-    			platform: 'ios',
-    			device: 'iphoneX',
-    		},
-    	}}
-    />
-    ```
-
-    > Note: **analyticsConfig** prop will only work when **analytics={true}**
+-   **appbaseConfig** `Object` [optional]
+    allows you to customize the analytics experience when appbase.io is used as a backend.
+    Read more about it over [here](/docs/reactivesearch/v3/advanced/analytics/#configure-the-analytics-experience).
```

Comment:
@ShahAnuj2610 We need to define the properties here, not all properties are present here.